### ISSUE TITLE
Issue with finding the right model

### ISSUE THREAD

**Issue by rithvikrthampy** [Samjith888/Keras-retinanet-Training-on-custom-datasets-for-Object-Detection-#5]
When should i stop the training?
is it when the loss is at the minimum and starts increasing in the next epoch
of should we take classification and regression loss as well?

**Comment by Samjith888** [comment]
> When should i stop the training?
> is it when the loss is at the minimum and starts increasing in the next epoch
> of should we take classification and regression loss as well?

You can calculate the mAP value by passing the following line into ur training command ( for this u need a val.csv file)
--compute-val-loss  --val-annotations val.csv

Also use the tensorboard which is already implemented here( use the tensorboard command in the tensorboard directory)
Then u can see the graph too
